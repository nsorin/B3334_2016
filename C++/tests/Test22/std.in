S seg1 0 20 10 0
R rect1 0 200 100 10
R rect2 0 10 10 0
OR reu1 rect1 rect2 seg1
R r1 5 10 10 5
R r2 7 20 20 7
OI r1int2 r1 r2
SAVE t1
CLEAR
R r1 0 10 10 0
SAVE t2
EXIT
