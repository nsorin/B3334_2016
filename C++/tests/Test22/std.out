#S seg1 0 0 10 20
OK
#R rect1 0 10 100 200
OK
#R rect2 0 0 10 10
OK
#OR reu1 rect1 rect2 seg1
OK
#R r1 5 5 10 10
OK
#R r2 7 7 20 20
OK
#OI r1int2 r1 r2
OK
#SAVE t1
OK
#CLEAR
OK
#R r1 0 0 10 10
OK
#SAVE t2
OK
#EXIT
